packages/mui-codemod/src/deprecations/circular-progress-classes/test-cases/expected.js

```diff
@@ -0,0 +1,6 @@
+import { circularProgressClasses } from '@mui/material/CircularProgress';
+
+('& .CircularProgress-circleDeterminate');
+('& .CircularProgress-circleIndeterminate');
```

Comment:
Shouldn't this have changed?

Reply:
fixed here. https://github.com/mui/material-ui/pull/42076/commits/b709f7c90b5b908a2f69bc9a805154e7af79b6d4 `Mui` was missing